contracts/staking/stakeManager/StakeManager.sol

```diff
@@ -498,6 +508,7 @@ contract StakeManager is IStakeManager, StakeManagerStorage {
         delete signerToValidator[validators[validatorId].signer];
         signerToValidator[_signer] = validatorId;
         validators[validatorId].signer = _signer;
+        latestSignerUpdateEpoch[validatorId] = latestSignerUpdateEpoch[validatorId].add(signerUpdateLimit);
```

Comment:
Please don't add it here, because if we update `signerUpdateLimit` then most of the validators will stay in the older limit.

Reply:
@0xAshish should I not initialize this?